project/settings.py

```diff
@@ -230,6 +230,11 @@
     MIDDLEWARE.append(
         'rollbar.contrib.django.middleware.RollbarNotifierMiddlewareExcluding404')
 
+CSP_CONNECT_SRC = [
+    "'self'",
+    "https://geosearch.planninglabs.nyc"

```

Comment:
Hmm, this could get out-of-sync with the actual web service we use to perform the geo search but um, at least the progressive enhancement error boundary will kick in and it will downgrade to baseline. Also, it is extremely unlikely to happen.